shared/common-adapters/choice-screen.desktop.js

```diff
@@ -0,0 +1,101 @@
+// @flow
+import React from 'react'
+import {Box, Text, Icon, Button} from '../common-adapters'
+import {globalStyles, globalColors, globalMargins} from '../styles/style-guide'
+import StandardScreen from './standard-screen'
+import type {Props} from './choice-screen'
+
+const ChoiceScreen = (props: Props) => {
+  const {options, onCancel, title} = {...props}
+  return (
+    <StandardScreen onClose={onCancel}>
+      <style>{rawCSS}</style>
+      <Text style={styleTitle} type='Header'>{title}</Text>
+      {options.map((op, idx) => (
+        <Box style={styleEntry} key={idx} className='choice-screen__entry' onClick={() => op.onClick()}>
+          <Box style={styleIconContainer} className='choice-screen__entry__icon-ctnr'>
+            {typeof op.icon === 'string' ? <Icon style={styleIcon} type={op.icon} className='choice-screen__entry__icon-ctnr__icon' /> : <Box style={styleIcon} className='choice-screen__entry__icon-ctnr__icon'>{op.icon}</Box>}
```

Comment:
Good call. I have line wrapping on so didn't notice, but I'll truncate it at ~80 or so